lib/public/views/Environments/Details/EnvironmentDetailsPage.js

```diff
@@ -22,11 +22,39 @@ import { frontLink } from '../../../components/common/navigation/frontLink.js';
 import { tabbedPanelComponent } from '../../../components/TabbedPanel/tabbedPanelComponent.js';
 import { ENVIRONMENT_DETAILS_PANELS_KEYS } from './EnvironmentDetailsModel.js';
 import { logsActiveColumns } from '../../Logs/ActiveColumns/logsActiveColumns.js';
-import { markdownDisplay } from '../../../components/common/markdown/markdown.js';
 import { PanelComponent } from '../../../components/common/panel/PanelComponent.js';
 import { aliEcsEnvironmentLinkComponent } from '../../../components/common/externalLinks/aliEcsEnvironmentLinkComponent.js';
 import { epnInfologgerLinkComponent, flpInfologgerLinkComponent } from '../../../components/common/externalLinks/infologgerLinksComponents.js';
 
+/**
+ * Display the raw environment configuration as a table
+ *
+ * @param {string} rawConfiguration the raw environment configuration to display, expected to be a JSON of string keys and values
+ * @return {Component} the resulting component
```

Comment:
I think you expect simply JSON, not JSON of string keys and values

Reply:
Correct, applied